src/main/ruby/rack/handler/servlet/default_env.rb

```diff
@@ -279,7 +279,7 @@ def servlet_context
             @servlet_env.servlet_context # ServletRequest#getServletContext()
           else
             if @servlet_env.respond_to?(:context) &&
-                @servlet_env.context.is_a?(javax.servlet.ServletContext)
+                @servlet_env.context.is_a?(jakarta.servlet.ServletContext)
```

Comment:
won't work, JRuby only provides some common names at top level such as `java`, `javax`, `org`.

so you'll need to prefix e.g. `Java::jakarta.servlet.ServletContext`.

p.s. `Java::JakartaServlet::ServletContext` (above) is okay.

Reply:
Hmm, interesting that no tests fail. Probably because above this line is

```ruby
if @servlet_env.respond_to?(:servlet_context) # @since Servlet 3.0
            @servlet_env.servlet_context # ServletRequest#getServletContext()
else
```

So this clause itself was a <= Servlet 2.5 backward compat. Maybe can be removed?

In an ideal world we'd be able to run tests across multiple servlet API versions to test compatibility more thoroughly...

In any case I should probably change all the references to use the same style, as there are too many combinations here.